### DISCUSSION TITLE
Using hops in clusters or as userobjects

### DISCUSSION THREAD

**Discussion by Slackking1** [mcneel/compute.rhino3d#577] [General]
Hi
Not sure if this is the right place to put this, but seems like it. 
I've been playing around with the hops component. I'm looking for a "low-code" solution to use hops definitions amongst colleagues, where they are up to date. 
I'm aware you can code everything in a plugin and make it fast, but i believe a grasshopper company environment should consist of proper plugins and shared grasshopper scripts. This leads me to the challenge. 

I've been trying to setup shared hops components. To make it easy for colleagues to access i want to get them as components in the ribbon. This works fine as long as i work with locale files. But when i point my hops components to a public GitHub url i get errors, for any other url aswell. 

Basically it seems i cant use hops components pointing to a GitHub repository url, in either clusters or .ghuser. i get a proxy mistake. i can assign the url directly in the component every time, but that a shitty workflow. From my research i found GhPython and flask. But the point of this is to make a shared component library amoungs colleagues that are not developers. 

I Tried multiple solutions all giving me the same error. I think the problem in it's essense is i cant use hops with urls in either clusters or userObjects, can this be true? 

Hope somebody can help a bit
